pkg/dockerregistry/server/context.go

```diff
@@ -0,0 +1,17 @@
+package server
+
+import "context"
+
+type contextKey string
+
+var registryClientKey contextKey = "registryClient"
```

Comment:
Historically, these things are stored in the `util.go`. Please either move context variables from `util.go` there or move this variables there.